styles/main.css

```diff
@@ -0,0 +1,968 @@
+:root {
+    --light-green: #3CDBAD;
+    --orangish: #FA9E07;
+    --white90: #FFFFFF90;
+    --gray-background: #F3F3F3
+    /*--highlight-color: #ff1b68;*/
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html {
+    height: 100%;
+    scroll-behavior: smooth;
+}
+
+body {
+    font-family: "et-line", "PT Sans", sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+.main-banner {
+    background: url("../images/home-bg.jpg") center/cover no-repeat border-box;
+    width: 100vw;
+    height: 50vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0;
+    padding: 0;
+    transition: background-image 1s ease-in-out;
+}
+
+.main-banner__content {
+    display: inline-block;
+    text-align: center;
+    color: white;
+}
+
+.main-banner__title {
+    font-size: 2rem;
+}
+
+.main-banner__description {
+    font-family: sans-serif;
+    font-size: 1.1rem;
+}
+
+.main-banner a {
+    display: inline-block;
+    text-decoration: none;
+    color: white;
+    font-family: "Lato", sans-serif;
+    padding: 10px 30px;
+    margin: 30px 10px;
+    border-radius: 4px;
+    font-size: 0.8rem;
+    font-weight: bold;
+}
+
+#main-banner__learn-more {
+    letter-spacing: 0.05rem;
+    border: 2px solid white;
+}
+
+#main-banner__contact {
+    letter-spacing: 0.05rem;
+    display: none;
+}
+
+.main-banner a:hover {
+    color: var(--light-green);
+    background-color: white;
+    border: 2px solid white;
+}
+
+@media(min-width: 550px) {
+    .main-banner__content {
+        padding: 3rem;
+    }
+    .main-banner__title {
+        font-size: 2.5rem;
+    }
+    .main-banner__description {
+        font-family: sans-serif;
+        font-size: 1.3rem;
+    }
+    #main-banner__contact {
+        display: initial;
+        border: 2px solid var(--light-green);
+        background-color: var(--light-green);
+    }
+    #main-banner__contact:hover {
+        color: var(--light-green);
+        background-color: white;
+        border: 2px solid white;
+    }
+    #main-banner__contact:focus,
+    #main-banner__contact:active {
+        color: white;
+        border: 2px solid var(--orangish);
+        background-color: var(--orangish);
+    }
+}
+
+@media(min-width: 800px) {
+    .main-banner__title {
+        font-size: 3.2rem;
+    }
+    .main-banner {
+        height: 100vh;
+    }
+    .main-banner__content {
+        max-width: 70%;
+    }
+}
+
+.main-header {
+    margin: 0;
+    display: flex;
+    position: sticky;
+    top: 0;
+    flex-direction: column;
+    align-items: center;
+    background: white;
+    box-shadow: 1px 1px 1px 1px lightgray;
+    z-index: 100;
```

Comment:
use values between -1 and 10, 10 being the highest value (you are never gonna have anything over it)